### DISCUSSION TITLE
Is there a way to select workspace based on where my mouse is?

### DISCUSSION THREAD

**Discussion by reikdas** [regolith-linux/regolith-desktop#969] [Q&A] [answered]
On default i3, the workspace where my mouse is is selected. But here, I have to click with my mouse to select the workspace that it is in.

**Comment by aacero** [comment, marked as answer]
I think you probably want "focus_follows_mouse yes" somewhere in your personal configuration files; I put it in ${HOME}/.config/regolith3/i3/config.d/i3-wm.conf:

```
$ cat .config/regolith3/i3/config.d/i3-wm.conf 
focus_follows_mouse yes
```

I'm using Regolith 3.1 on Ubuntu 23.10 (mantic), so your specific config location may differ.  I've carried this setting forward since the early Regolith days so I'm pretty it will work on whichever version of Regolith you're using.  After you've added this, you'll need to reload the config with Win+Shift+R (or restart) for it to take effect.

Edit: in case it matters I have a symbolic link from i3 -> wm in .config/regolith3 so the paths ".config/regolith3/i3/config.d/" and ".config/regolith3/wm/config.d" are interchangable for me.
